### DISCUSSION TITLE
Unexpected creation of lithium in low metallicities

### DISCUSSION THREAD

**Discussion by almendradelmoral** [MESAHub/mesa#584] [Q&A]
Hello,

I'm having issues when i use MESA at low metallicities. It is creating an unexpected creation of lithium at low metallicities and this doesn't happen at solar metallicity. The inlist that i'm using are from "A helium-flash-induced mixing event can explain the lithium abundances of red clump stars" which is in this [link](https://zenodo.org/record/4688026).  This behaviour can be seen in the attached figure
![example](https://github.com/MESAHub/mesa/assets/43151058/77bd04f2-24e3-442c-b4dd-316429495492)

Also, I wanted to know how i can set an initial Lithium abundance for different masses.

**Comment by evbauer** [comment]
I'd recommend posting this sort of thing to the [mesa-users mailing list](https://lists.mesastar.org/mailman/listinfo/mesa-users), as there's a much broader community of people subscribed there who will be notified and may be able to help.

**Reply by almendradelmoral** [reply]
thanks!
